.github/workflows/docs.yaml

```diff
@@ -10,9 +10,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
-      - run: npm i
-      - run: npm i ink-docstrap jsdoc
+        uses: actions/checkout@v3
+      - run: npm ci
+      - run: npm i ink-docstrap jsdoc taffydb
```

Comment:
I'm curious, would taffydb be an optional dependency for jsdoc and ink-docstrap?

Reply:
Its supposed to be, but the docs generation & deploy fails without it.